packages/esm-ward-app/src/ward-patient-workspace/ward-patient.workspace.tsx

```diff
@@ -2,24 +2,23 @@ import React, { useEffect, useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 import { InlineLoading, InlineNotification } from '@carbon/react';
 import {
+  age,
+  attach,
   type DefaultWorkspaceProps,
   ExtensionSlot,
-  attach,
   getPatientName,
   usePatient,
-  age,
 } from '@openmrs/esm-framework';
 import styles from './ward-patient.style.scss';
+import { type WardPatientCardProps } from '../types';
 
 attach('ward-patient-workspace-header-slot', 'patient-vitals-info');
 
-export interface WardPatientWorkspaceProps extends DefaultWorkspaceProps {
-  patientUuid: string;
-}
+interface WardPatientWorkspaceProps extends DefaultWorkspaceProps, WardPatientCardProps {}
 
-export default function WardPatientWorkspace({ patientUuid, setTitle }: WardPatientWorkspaceProps) {
+export default function WardPatientWorkspace({ setTitle, patient: { uuid } }: WardPatientWorkspaceProps) {
   const { t } = useTranslation();
-  const { patient, isLoading, error } = usePatient(patientUuid);
+  const { patient, isLoading, error } = usePatient(uuid);
```

Comment:
What are we getting from the `patient` prop, that we are re-fetching the patient here?
Thanks!